corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -2,7 +2,7 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-{% block head %}{{ block.super }}
+{% block js %}{{ block.super }}
```

Comment:
oh yeah! It crossed my mind and then slipped in the excitement of fixing it.

So with this reverted, the bug is back. What's a better solution for this?